src/PopEngine/index.ts

```diff
@@ -211,37 +211,35 @@ export class PopEngine {
       this.setState(pop, PopStateType.PRE_HIDE, pop.opts, null, false);
 
       this._timeouts.timeToHoverOnPop[groupId] = setTimeout(function(): void {
-        // hide children
-        if (!!pop.childPops.length) {
-          pop.childPops.forEach(function(child: Pop): void {
-            this.hidePop(child.targetEl, hideFullChain);
-          }, this);
-        }
 
-        // hide pop
-        let popOver = <Element>document.querySelector('div[pop-id="' + groupId + '"]');
-        targetElement.removeAttribute('pinned-pop');
+        let popChain = popChainManager.getFullPopChain(pop, hideFullChain);
 
-        this._maybeClearHandler(this._handlers[groupId]);
-        this._listenForScroll(false, groupId, popOver);
+        popChain.forEach(function(p: Pop): void {
+          if (!!p.parentPop) {
+            let index = p.parentPop.childPops.indexOf(p);
```

Comment:
```
 +            let index = p.parentPop.childPops.indexOf(p);
 +            if (index !== -1) {
 +              p.parentPop.childPops.splice(index, 1);
 +            }
 +            p.parentPop = null;
```

Can we make this into its own function? Seems handy, maybe called `removeRelationshipWithParent` or something of that sort.